{{cookiecutter.app_name}}/Procfile

```diff
@@ -1 +1,2 @@
+release: flask db upgrade
```

Comment:
I haven't used Heroku before, so basing this question from a cursory reading of the docs, but: does adding this to the Procfile result in the creation of additional dynos when starting up the app? Would it preferable to have the user run the DB upgrade in a one-off Dyno at the same time that they run `flask db migrate`?

Reply:
I am not sure did I understand you good. What do you mean by running DB upgrade in a one-off Dyno together with `flask db migrate`? As I understood, did you think about adding line like: `release: flask db migrate && flask db upgrade` ? If yes, it will not work due to the fact that `migrations` folder created on Heroku filesystem will be erased (look comment above, additional source: https://devcenter.heroku.com/articles/dynos#ephemeral-filesystem). Actually it will work at the first time, but during next release/dyno_restart exception about missing migrations will be raised.

Reply:
Sorry for the confusion; I didn't realize that the process name `release` is special in Heroku & runs as a one-off dyno by default. What you have looks correct!